api/src/opentrons/protocol_api/protocol_context.py

```diff
@@ -825,29 +846,32 @@ def load_instrument(
         tip_racks: Optional[List[Labware]] = None,
         replace: bool = False,
     ) -> InstrumentContext:
-        """Load a specific instrument required by the protocol.
+        """Load a specific instrument for use in the protocol.
+
+        When analyzing the protocol on the robot, instruments loaded with this method
+        are compared against the instruments attached to the robot. You won't be able to
+        start the protocol until the correct instruments are attached and calibrated.
```

Comment:
Consider "compared to" instead of "compared with." Then, change the next sentence to "You cannot start the protocol ..."

```suggestion
        When analyzing the protocol on the robot, instruments loaded with this method
        are compared to the instruments attached to the robot. You cannot
        start the protocol until the correct instruments are attached and calibrated.
```

Reply:
It's subtle, but I think the tense here makes a useful distinction. "When doing X…you cannot Y" makes X and Y sound contemporaneous. But analysis always precedes the user starting the protocol.